boards/phytec/phyboard_nash/doc/index.rst

```diff
@@ -0,0 +1,178 @@
+.. _phyboard_nash:
+
+PhyBOARD Nash (NXP i.MX93)
```

Comment:
```suggestion
phyBOARD-Nash (NXP i.MX93)
```

Reply:
As you can see looks a little bit odd here: https://docs.zephyrproject.org/latest/boards/phytec/index.html

You could also just called it `phyBOARD-Nash i.MX93`